CRM/Import/Form/DataSource.php

```diff
@@ -150,7 +150,7 @@ protected function submitFileForMapping($parserClassName, $entity = NULL) {
     $parser->setMaxLinesToProcess(100);
     $parser->run($fileName,
       $separator,
-      $mapper,
+      [],
```

Comment:
Looks like you could also delete the now-unused variable `$mapper`

Reply:
@colemanw not quite yet 

![image](https://user-images.githubusercontent.com/336308/162355542-8223af02-cc30-4b58-bea2-c0ce03a552f6.png)